templates/_helpers.yaml

```diff
@@ -53,15 +53,21 @@
 
 {{/* User defined Airflow environment variables */}}
 {{- define "custom_airflow_environment" }}
-  # Dynamically created environment variables
-  {{- range $i, $config := .Values.env }}
-  - name: {{ $config.name }}
-    value: {{ $config.value | quote }}
-    {{- if eq $.Values.executor "KubernetesExecutor" }}
-  - name: AIRFLOW__KUBERNETES_ENVIRONMENT_VARIABLES__{{ $config.name }}
-    value: {{ $config.value | quote }}
-    {{- end }}
-  {{- end }}
+{{- if .Values.mountAllFromSecretName }}
+envFrom:
+{{- include "custom_airflow_environment_mount_all" . }}
+env:
+{{- include "custom_airflow_environment_config" . }}
+{{- include "standard_airflow_environment" . }}
+{{- else }}
+env:
+{{- include "custom_airflow_environment_mount_single" . }}
+{{- include "custom_airflow_environment_config" . }}
+{{- include "standard_airflow_environment" . }}
+{{- end -}}
+{{- end -}}
+
+{{- define "custom_airflow_environment_mount_single" }}
```

Comment:
@kushalmalani what do you think about combining the parts that loop and combine `.Values.secret` and `.Values.env` into a single helper called `custom_airflow_environment_mount_individual`? Slightly different, but maybe a little clearer to the reader?

This would essentially combine `custom_airflow_environment_mount_single` and `custom_airflow_environment_config` into a single helper.

Once these are combined, I think we could simplify the logic to say:

```
{{- if .Values.mountAllFromSecretName }}
envFrom:
{{- include "custom_airflow_environment_mount_all" . }}
env:
{{- include "standard_airflow_environment" . }}
{{- else }}
env:
{{- include "custom_airflow_environment_mount_individual" . }}
{{- include "standard_airflow_environment" . }}
{{- end -}}
{{- end -}}
```

This essentially makes it so that if you are using the new method, the old methods are completely ignored. If not, then it should behave exactly how it did before. What do you think?